lua/nui/popup/border.lua

```diff
@@ -384,6 +384,34 @@ function Border:resize()
   end
 end
 
+function Border:reposition()
```

Comment:
realized I didn't make Border:reposition accept new relative value

Reply:
Border's method should not be called directly. So this method is not supposed to take any arguments.
Border should reposition itself based on the configuration of Popup.